application-accelerator/configuration.hbs.md

```diff
@@ -7,6 +7,26 @@ Application Accelerator pulls content from accelerator source repositories using
 
 If the repository used is accessible anonymously from a public server, then you do not have to configure anything additional. Accelerators are created either using the Tanzu CLI or by applying a YAML manifest using `kubectl`.
 
+## <a id="using-git-ops"></a> Using a Git-Ops style configuration for deploying a set of managed accelerators
+
+In order to enable a Git-Ops style of managing resources used for deploying accelerators there is a new set of properties for the App Accelerator configuration. The resources will be managed using a Carvel kapp-controller App that watches a Git repository containing the mainifests for the accelerators. This means that you can make changes to the manifests, or to the accelerators they point to, and the changes will be reconciled and reflected in the deployed resources.
+
+You can specify the following accelerator configuration properties when installing the Application Accelerator. The same properties can be provided in the `accelerator` section of the `tap-values.yaml` file:
+
+```yaml
+managed_resources:
+  enable: true
+  git:
+    url: GIT-REPO-URL
+    ref: origin/main
+    sub_path: null
+    secret_ref: git-credentials
+```
+
+Where:
+
+- `GIT-REPO-URL` is the URL of a Git repository that contains manifest YAML files for the accelerators that you want to have managed (see below for manifest examples). You can specify a `sub_path` if necessary and also a `secret_ref` if the repository requires authentication. If not needed, then leave these additional properties out.
```

Comment:
Quick suggestion - is it possible to have a hyperlink to the docs starting on line 128 so a user has an intuitive way to jump to the relevant piece of info on this?

Reply:
Added this